package.json

```diff
@@ -9,6 +9,7 @@
   "scripts": {
     "create-workspace": "community-cli workspace create",
     "postinstall": "husky",
+    "lint-staged": "lint-staged",
```

Comment:
This this one might be good to separate out in a new PR? Always good to have repo-wide changes like this separate from other additions